cli/src/main/java/org/jboss/as/cli/impl/aesh/cmd/security/model/ElytronUtil.java

```diff
@@ -488,6 +490,7 @@ public static ServerSSLContext getServerSSLContext(CommandContext context, Strin
         if (sslContext != null) {
             String kmName = sslContext.get(Util.KEY_MANAGER).asString();
             ModelNode km = getChildResource(kmName, Util.KEY_MANAGER, context);
+            checkNotNullParamWithNullPointerException("ModelNode for " + kmName, km);
```

Comment:
The `"ModelNode for " + kmName` text ends up in the [following log](https://github.com/wildfly/wildfly-common/blob/main/src/main/java/org/wildfly/common/_private/CommonMessages.java#L62)

So, this text should be a parameter name instead. Notice that messages of the loggers are translated into different languages. What about Util.KEY_MANAGER?

The other points look fine to me, CC @jfdenise

Reply:
I would suggest to use a NullPointerException (compatible Exception, same as before, else a IllegalStateException) with a custom text here, because Util.KEY_MANAGER (kmName) itself is not null, but the childResource.

Reply:
That sounds like an alternative, although I cannot suggest to you how you can internationalize it, I searched for a valid logger but I'm not sure how this is treated on CLI. In general, if a message arrives at the user, it should be internationalized.

Reply:
@yersan The method is not in use currently. Several other exceptions use hardcoded English texts (see some lines above), I have prepared the following:
```
            if (km == null) {
                throw new NullPointerException("The ModelNode of '" + kmName + "' must not be null");
            }
```

Reply:
ok @boris-unckel , thanks, I've approved it from my side, but this should be approved by the component lead, @jfdenise FYI, not sure if you are interested in adding these changes.